examples/gan/dcgan_tf_keras/determined_model_def.py

```diff
@@ -0,0 +1,59 @@
+"""
```

Comment:
fix: can you change the name of this file to model_def.py to keep consistency?

Reply:
done